nvflare/app_common/executors/client_api_launcher_executor.py

```diff
@@ -122,6 +123,7 @@ def prepare_config_for_launch(self, fl_ctx: FLContext):
                 ConfigKey.CLASS_NAME: pipe_export_class,
                 ConfigKey.ARG: pipe_export_args,
             },
+            ConfigKey.HEARTBEAT_TIMEOUT: self._heartbeat_timeout,
```

Comment:
self.heartbeat_timeout is in the very base class TaskExchanger, should we reuse that instead?